### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by nourryan**
# Updating an extension

You can delete this whole template and just describe which extension is updated and optionally tell us in a sentence what has been changed.

To make extension updates easier in the future, you may consider replacing specific git hash in your s4ext file by a branch name (for example: `master` or `release` for nightly version; `(majorVersion).(minorVersion)` such as `4.10` for stable Slicer version).

# Submitting a new extension

Thank you very much for considering sharing your extension with the Slicer community.

To make sure users can find your extension, understand what it is intended for and how to use it, please complete the checklist below. You do not need to complete all the item by the time you submit the pull request, but most likely the changes will only be merged if all the tasks are done.

- [x] Extension has a reasonable name (not too general, not too narrow, suggests what the extension is for)
- [x] Repository name is Slicer+ExtensionName
- [x] Extension description summarizes in 1-2 sentences what the extension is usable (should be understandable for non-experts)
- [x] Extension URL and revision (scmurl, scmrevision) is correct, consider using a branch name (master, release, ...) instead of a specific git has to avoid re-submitting pull request whenever the extension is updated
- [x] Extension icon URL is correct
- [x] Screenshot URLs (screenshoturls) are correct, contains at least one
- [x] Homepage URL points to valid webpage containing the following:
  - [x] Extension name
  - [x] Short description: 1-2 sentences, which summarizes what the extension is usable for
  - [x] At least one nice, informative image, that illustrates what the extension can do. It may be a screenshot.
  - [x] Description of contained modules: at one sentence for each module
  - [x] Tutorial: step-by-step description of at least the most typical use case, include a few screenshots, provide download links to sample input data set

See more information about the submission process on the Slicer wiki: https://www.slicer.org/wiki/Documentation/Nightly/Developers/Tutorials/BuildTestPackageDistributeExtensions

If you have any questions or comments then please describe them here.

### CODE REVIEW

**Comment by jcfr** [comment]
@nourryan Thanks for your contribution, before we move forward with the process, could you go through the checklist above ?

**Comment by nourryan** [comment]
is it ok for you ?

**Comment by FredericBr** [comment]
Dear Jean-Christophe,
Since we upload files to suit our checklist, do you have some issue about our extension (Geodesic Slicer)?

File: GeodesicSlicer.s4ext [hunk 1]
```diff
@@ -18,7 +18,7 @@ depends NA
 build_subdirectory .
 
 # homepage
-homepage http://slicer.org/slicerWiki/index.php/Documentation/Nightly/Extensions/GeodesicSlicer
+homepage https://www.slicer.org/wiki/Documentation/4.8/Modules/GeodesicSlicer
```
Comment (jcfr) [inline GeodesicSlicer.s4ext]: Looks great.

Could you also duplicate the content into the following pages:
* https://www.slicer.org/wiki/Documentation/4.10/Modules/GeodesicSlicer/edit
* https://www.slicer.org/wiki/Documentation/Nightly/Modules/GeodesicSlicer/edit

File: GeodesicSlicer.s4ext [hunk 2]
```diff
@@ -28,18 +28,18 @@ contributors Frederic Briend ((UNICAEN)), Antoine Nourry ((UMS 3408))
...
+iconurl https://www.slicer.org/w/images/d/dc/GeodesicSlicer_logo.png
 
 # Give people an idea what to expect from this code
 #  - Is it just a test or something you stand behind?
 status 
 
 # One line stating what the module does
 description This module calculates geodesic path in 3D structure. Thanks to this geodesic path, this module could draw an EEG 10-20 system
-<li>See <a href=
+<li>See <a href='https://www.slicer.org/wiki/Documentation/4.8/Modules/GeodesicSlicer'>Geodesic Slicer documentation</a> for more details.</li>
```
Comment (jcfr) [inline GeodesicSlicer.s4ext]: This should be on the same line as the previous sentence.

File: GeodesicSlicer.s4ext [hunk 3]
```diff
@@ -32,7 +32,7 @@ iconurl https://www.slicer.org/w/images/d/dc/GeodesicSlicer_logo.png
 
 # Give people an idea what to expect from this code
 #  - Is it just a test or something you stand behind?
-status 
+Maintained code
```
Comment (jcfr) [inline GeodesicSlicer.s4ext]: this should be `status  Maintained code`. The keyword `status` is useful to describe the value.

**Comment by FredericBr** [comment]
Hi Jean-Christophe, Is-it ok now?

**Comment by FredericBr** [comment]
Hi Jean-christophe,
Forgive me to hurry you, but we would like the extension to be released before we send our article for revisions.
Since our last correction, does everything seem correct?

File: GeodesicSlicer.s4ext [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+# url to icon (png, size 128x128 pixels)
+iconurl https://www.slicer.org/w/images/d/dc/GeodesicSlicer_logo.png
+
+# Give people an idea what to expect from this code
+#  - Is it just a test or something you stand behind?
+status Maintained code 
+
+# One line stating what the module does
+description This module calculates geodesic path in 3D structure. Thanks to this geodesic path, this module could draw an EEG 10-20 system
+<li>See <a href='https://www.slicer.org/wiki/Documentation/Nightly/Modules/GeodesicSlicer'>Geodesic Slicer documentation</a> for more details.</li>
```
Comment (jcfr) [inline GeodesicSlicer.s4ext]: description should be on only one line

**Comment by jcfr** [comment]
@FredericBr  Should the extension be named simply ` Geodesic` instead of ` GeodesicSlicer` ? 

Cc: @lassoan

**Comment by jcfr** [comment]
> Should the extension be named simply Geodesic instead of GeodesicSlicer

we can revisit later.

**Comment by jcfr** [comment]
I fixed the misc issue and submitted an other PR. See https://github.com/Slicer/ExtensionsIndex/pull/1608

**Pull request closed without merging**